### DISCUSSION TITLE
Air Purifier Entity

### DISCUSSION THREAD

**Discussion by CodeFatherG** [home-assistant/architecture#1009] [Entity Models] [answered]
Hi!

I moved to home assistant because google home had trash integration for my samsung air purifier through smartthings. To my disappointment I found that smartthings integration is stale and not maintained and the air purifier integration was poor. I have since gone and fixed this integration myself and realised, why isn't there an air purifier entity?

After covid air purifiers are very very common in peoples homes. In essence, an air purifier is a fan and a air quality entity at its core. I think that making an air purifier entity leveraging the air quality entity and fan entity would be a step towards standardising the air purifier integrations

**Comment by MartinHjelmare** [comment, marked as answer]
Why isn't the fan entity plus sensor entities for the measurements enough? We're moving away from complex entities with multiple features.

**Reply by frenck** [reply]
I agree with that. We are currently perfectly capable of representing an air purifier in our current architecture.
